.azure-pipelines/MacOS-CI.yml

```diff
@@ -30,12 +30,26 @@ jobs:
       versionSpec: '$(python.version)'
 
   - script: |
-      git submodule update --init --recursive
-      python -m pip install --upgrade setuptools
-      python -m pip install numpy
-      conda install -y -c conda-forge pybind11 protobuf
```

Comment:
I noticed that pybind11 was removed, is this no longer necessary here?

Reply:
Yes, actually ONNX uses pybind11 as a submodule so basically it should not need to manually install pybind11 for ONNX